Documentation~/Installation.md

```diff
@@ -1,6 +1,23 @@
 # Installation
 
-## Automatic Installation
+## Unity Project Setup via DCC tools
+We can install MeshSync on a Unity project through DCC tools. This install method is designed for users who are not familiar with the Unity Editor or the Unity Package Manager.
+
+Currently, this install method is available only for Blender. For other DCC tools, see [MeshSync](https://docs.unity3d.com/Packages/com.unity.meshsync@latest)'s installation page.
+### Blender
+Install the Blender add-on:
+1. Download the latest MeshSyncDCCPlugins release from [here](https://github.com/Unity-Technologies/MeshSyncDCCPlugins/releases).
+1. Install the add-on manually, as described [here](./Installation.md#blender-1).
```

Comment:
Might be better if we are telling the users to reuse the zip files from step 1 directly instead of telling them open `Packages/MeshSync DCC Plugins/Editor/Plugins~` which might be confusing.

Approving this for now though.